tensorflow/tools/graph_transforms/transform_graph.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Exposes the Python wrapper for graph transforms over GraphDef protocol
+buffer messages."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.core.framework import graph_pb2
+from tensorflow.python.pywrap_tensorflow import TransformGraphWithStringInputs
+from tensorflow.python.util import compat
```

Comment:
`third_party/tensorflow/tools/graph_transforms/transform_graph.py", line 24, in <module>
    from google3.third_party.tensorflow.python.pywrap_tensorflow import TransformGraphWithStringInputs
ImportError: cannot import name 'TransformGraphWithStringInputs'`

@frreiss can you please fix this error

Reply:
This should be fixed now.